public/js/navigator-js/integration/BackboneRecipe.js

```diff
@@ -0,0 +1,43 @@
+var BackboneRecipe = {
```

Comment:
@bassettsj es6 will be in a follow up PR.

@elimak wanted me (and I agree) to start with just the features so as to keep the diff smaller and make the functional changes more easy to distinguish.